setup.py

```diff
@@ -93,60 +93,4 @@ def run(self):
         'build_assets': WebpackBuildCommand,
         'watch': WebpackDevelopCommand,
     },
-    zip_safe=False,
-    packages=['sphinx_rtd_theme'],
-    package_data={'sphinx_rtd_theme': [
-        'theme.conf',
-        '*.html',
-        'static/css/*.css',
-        'static/css/fonts/*.*',
-        'static/js/*.js',
-    ]},
-    include_package_data=True,
-    # See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
-    entry_points = {
-        'sphinx.html_themes': [
-            'sphinx_rtd_theme = sphinx_rtd_theme',
-        ]
-    },
-    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
```

Comment:
Great to get rid of this btw.

Reply:
I take that back. This option is entirely great and we should keep having it in setup.cfg. So I've added it again. This should also ensure that we don't have regressions from #1116